crates/core/src/types/function.rs

```diff
@@ -1,5 +1,4 @@
 //! Adapted from <https://github.com/YarnSpinnerTool/YarnSpinner/blob/da39c7195107d8211f21c263e4084f773b84eaff/YarnSpinner/Types/FunctionType.cs>
-#[cfg(any(feature = "bevy", feature = "serde"))]
```

Comment:
Does this still work when compiling only the crate `yarnspinner` without the bevy feature, but with the serde feature?

Reply:
Yes, I ran with --no-default-features --features serde to test

The actual feature gating was/is already being done the prelude

pub mod prelude {
    #[cfg(any(feature = "bevy", feature = "serde"))]
    pub use crate::feature_gates::*;
    // ...
}